src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -662,11 +662,16 @@
   - name: --subnet-prefix
     type: string
     short-summary: The CIDR used on the Subnet into which to deploy the cluster.
+  - name: --customer-admin-group-id
+    type: string
+    short-summary: The ID of an Azure Active Directory Group that memberships will get synced into the OpenShift group "osa-customer-admins". If not specified, no cluster admin access will be granted.
 
 
 examples:
   - name: Create an OpenShift cluster and auto create an AAD Client
     text: az openshift create -g MyResourceGroup -n MyManagedCluster --fqdn {FQDN}
+  - name: Create an OpenShift cluster and auto create an AAD Client and setup cluster admin group
+    text: az openshift create -g MyResourceGroup -n MyManagedCluster --fqdn {FQDN} --customer-admin-group-id {GROUP_ID}
```

Comment:
This is going to be incompatible with PR #9083 which removes FQDN.

Reply:
@tjprescott FQDN is not being removed. Its being made optional. PR #9083 will be done on top of this PR.